WooCommerce/WooCommerceTests/ViewRelated/Orders/Simple Payments/SimplePaymentsSummaryViewModelTests.swift

```diff
@@ -89,16 +103,176 @@ final class SimplePaymentsSummaryViewModelTests: XCTestCase {
+                                                       taxAmount: "$4.3",
+                                                       taxLines: [],
+                                                       featureFlagService: featureFlagService)
+        // Then
+        XCTAssertFalse(viewModel.showTaxBreakup)
+    }
+
+    func test_showTaxBreakup_is_true_if_taxLinesInSimplePayments_feature_flag_is_on() {
+        // Given
+        let featureFlagService = MockFeatureFlagService(isTaxLinesInSimplePaymentsOn: true)
+        let viewModel = SimplePaymentsSummaryViewModel(providedAmount: "100",
+                                                       totalWithTaxes: "104.30",
+                                                       taxAmount: "$4.3",
+                                                       taxLines: [],
+                                                       featureFlagService: featureFlagService)
+        // Then
+        XCTAssertTrue(viewModel.showTaxBreakup)
+    }
+
+    func test_showChargeTaxesToggle_is_true_if_taxLinesInSimplePayments_feature_flag_is_off() {
+        // Given
+        let featureFlagService = MockFeatureFlagService(isTaxLinesInSimplePaymentsOn: false)
+        let viewModel = SimplePaymentsSummaryViewModel(providedAmount: "100",
+                                                       totalWithTaxes: "104.30",
+                                                       taxAmount: "$4.3",
+                                                       taxLines: [],
+                                                       featureFlagService: featureFlagService)
+        // Then
+        XCTAssertTrue(viewModel.showChargeTaxesToggle)
+    }
+
+    func test_when_taxLines_is_not_empty_and_taxLinesInSimplePayments_feature_flag_is_off_then_showChargeTaxesToggle_is_true() {
+        // Given
+        let featureFlagService = MockFeatureFlagService(isTaxLinesInSimplePaymentsOn: false)
+        let viewModel = SimplePaymentsSummaryViewModel(providedAmount: "100",
+                                                       totalWithTaxes: "104.30",
+                                                       taxAmount: "$4.3",
+                                                       taxLines: [
+                                                        SimplePaymentsSummaryViewModel.TaxLine(id: 1,
+                                                                                               title: "State",
+                                                                                               value: "4.23")
+                                                       ],
+                                                       featureFlagService: featureFlagService)
+        // Then
+        XCTAssertTrue(viewModel.showChargeTaxesToggle)
+    }
+
+    func test_when_taxLines_is_empty_and_taxLinesInSimplePayments_feature_flag_is_on_then_showChargeTaxesToggle_is_false() {
```

Comment:
Nit: Same comment as above - this can be renamed to be clearer:
```suggestion
    func test_showChargeTaxesToggle_is_false_when_taxLines_is_empty_and_taxLinesInSimplePayments_feature_flag_is_on() {
```

Reply:
Done in https://github.com/woocommerce/woocommerce-ios/pull/5943/commits/50f48dc47d31ae10c83b4727969eb2a89aba3fe3